lib/shared-class.js

```diff
@@ -78,15 +69,17 @@ function SharedClass(name, ctor, options) {
 }
 
 /**
- * Normalize HTTP path.
+ * Set additional options on this shared class, by merging the specified options
+ * into the already set options.
+ *
+ * @param options {Object} Set of options to be set on this shared class.
  */
 
-SharedClass.normalizeHttpPath = function(path) {
-  if (typeof path !== 'string') return;
-  return path.replace(/[^\/]+/g, function(match) {
-    if (match.indexOf(':') > -1) return match; // skip placeholders
-    return inflection.transform(match, ['underscore', 'dasherize']);
-  });
+SharedClass.prototype.setOptions = function(options) {
+  if (options) {
+    this.options = !this.options ? options :
+      extend(extend({}, this.options), options);
```

Comment:
I wasn't sure if it was OK to use `Object.assign()` or not. But that code is now gone anyway!